daemon/containerd/image.go

```diff
@@ -84,6 +84,13 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 	ociimage := presentImages[0]
 
 	img := dockerOciImageToDockerImagePartial(image.ID(desc.Target.Digest), ociimage)
+
+	parent, err := i.getImageLabelByDigest(ctx, desc.Target.Digest, imageLabelClassicBuilderParent)
+	if err != nil {
+		return nil, errdefs.System(fmt.Errorf("failed to determine Parent property: %w", err))
+	}
+	img.Parent = image.ID(parent)
```

Comment:
Bit on the fence; is this critical enough to make whole of `GetImage` fail?

Reply:
Probably not that critical, changed.